tests_thien/pages/dashboard/main_page.py

```diff
@@ -0,0 +1,10 @@
+from selenpy.element.base_element import BaseElement
+
+class DashBoardMainPage():
+    _title_page_name = BaseElement("xpath=//title")
+    
+    def __init__(self):
+        pass
+    
+    def check_main_page_appears(self):
+        return self._title_page_name.text == "TestArchitect ™ - Execution Dashboard"
```

Comment:
Get page title using the title property of web driver instead of creating new xpath

Reply:
updated